proto/money/rpc/card_processing_svc.proto

```diff
@@ -52,13 +52,9 @@ message PaymentCardVerificationRequest {
 }
 
 service CardProcessingSvc {
-	rpc GetAllPendingCaptures(.google.protobuf.Empty) 
-		returns (CardProcessingSvcCaptureEnumResponse);
-		
-	rpc GetAllPendingRefunds(.google.protobuf.Empty) 
```

Comment:
Removed this low-level stuff in favor of a single, unified "get open batches" endpoint, with dual "Get Closed Batches".